script/rtspServer.js

```diff
@@ -0,0 +1,15 @@
+const Stream = require('node-rtsp-stream');
+
+// Get node args
+const args = process.argv.slice(2);
+
+new Stream({
+  name: 'RTSP-Stream',
+  streamUrl: args[0],
+  wsPort: parseInt(args[1]),
+  ffmpegOptions: {
+    // options ffmpeg flags
+    '-r': 30, // options with required values specify the value after the key
```

Comment:
I am not sure if all cameras are at 30 fps, the value should be detected automatically or passed as an argument.

Reply:
I tested both our cameras and they both support 720p@30fps. I changed it so it uses the stream's native framerate using the `-re` flag.

Reply:
After testing the `-re` flag is just for inputs so it doesn't work for our case. I'll keep `-r 30` for now since all our cameras will be used at 30 fps.